Dockerfile

```diff
@@ -1,30 +1,30 @@
-FROM node:22-slim AS base
+FROM node:lts-alpine as builder
```

Comment:
how do you feel about lts instead of explicitly stating 22? (and how do you feel about alpine?)

Reply:
LTS is good 😊